integrations/python-agent-framework/autogen.mdx

```diff
@@ -1,12 +1,255 @@
 ---
 title: "AutoGen"
-description: "Learn how to integrate Membit with AutoGen."
+description: "Integrate Membit's real-time social context into your AutoGen agents and multi-agent systems using MCP tools."
 icon: "autogen"
 ---
 
-Welcome to the Membit and AutoGen integration guide. This section will provide you with all the information you need to connect Membit's real-time context to your AutoGen agents.
+Enhance your AutoGen applications with real-time social media context from Membit. This integration allows your AI agents to access up-to-the-minute discussions, trends, and insights from platforms like X (Twitter), Farcaster, and more.
 
-<Info>
-  This documentation is currently under development. Please check back soon for
-  more details.
-</Info>
+## Prerequisites
+
+Before you begin, ensure you have:
+
+- Python 3.10 or higher installed
+- A Membit account with API access
+- Basic familiarity with AutoGen agents
+- Node.js installed (for MCP remote client)
+
+<Warning>
+You'll need valid Membit API credentials and the MCP remote URL. If you don't have access yet, [get your API key](/access-and-auth) to get started.
+</Warning>
+
+## Installation
+
+Install the required packages for AutoGen and MCP integration:
+
+```bash
+# Install AutoGen with MCP tools
+pip install autogenstudio
+
+# Install MCP remote client (requires Node.js)
+npm install -g mcp-remote
+```
+
+<Tip>
+We recommend using a virtual environment to manage your Python dependencies and avoid conflicts.
+</Tip>
+
+## Quick Start
+
+<Steps>
+<Step title="Import required modules">
+Import the necessary components for AutoGen and MCP integration:
+
+```python
+import asyncio
+
+from autogen_agentchat.agents import AssistantAgent
+from autogen_agentchat.ui import Console
+from autogen_ext.models.openai import OpenAIChatCompletionClient
+from autogen_ext.tools.mcp import McpWorkbench, StdioServerParams
+```
+</Step>
+
+<Step title="Configure MCP server parameters">
+Set up the connection to Membit's MCP server:
+
+```python
+params = StdioServerParams(
+    command="npx",
+    args=[
+        "mcp-remote",
+        "https://mcp.membit.ai/mcp",
+        "--header",
+        "X-Membit-Api-Key:${API_KEY}",
+    ],
+    env={
+        "API_KEY": <your-api-key>,
```

Comment:
should it be MEMBIT_API_KEY, your-membit-api-key? (to align with https://github.com/bandprotocol/membit-mcp/pull/1/files). not sure if it's necessary.